images/inference-services/kfserving/run.sh

```diff
@@ -28,7 +28,7 @@ if [ "${PREDICTOR}" = "auto" ]; then
     fi
 
     PREDICTOR=$(mc cat "${model_bucket}"/MLmodel | awk -F '.' '/loader_module:/ {print $2}')
-    if [ "${PREDICTOR}" = "onnx" ]; then
+    if [[ "${PREDICTOR}" =~ "^(onnx|keras)$" ]]; then
         PREDICTOR="triton"
```

Comment:
I'll go back a little, even to the previous version of this condition: why are you serving method based on the model library? Shouldn't this be independent?

That said, shouldn't we really primary support selection of serving mechanism by user, instead of making these automatic assumptions?

Reply:
Because so far we have only tested onnx and keras serving with triton, even if other serving backends were being tested one should be chosen to be the default in case of `auto`.

Reply:
OK, although I might rather prefer examples where we show users how they can influence the worklfow with extra arguments...